README.md

```diff
@@ -9,11 +9,12 @@ produced from the Rust code using [Rust WASM](https://rustwasm.github.io/docs/bo
 The TypeScript library is experimental. Over time, it might replace the [bitbox02-api NPM
 package](https://www.npmjs.com/package/bitbox02-api).
 
-# Rust
+## Rust (required)
```

Comment:
i find these required/optional tags confusing. typescript is required if you want to do web stuff, for example.

why not just add a note at the top of the typescript section that says "In addition to the steps described above, ..."?